shared/actions/wallets.js

```diff
@@ -182,6 +179,10 @@ const navigateToAccount = (
     // Link existing failed, don't nav
     return
   }
+  return Saga.put(Route.navigateUp())
```

Comment:
Since we need the create new and link existing popups to return to the send form if triggered from the send form, I think the best way to handle this is to have them navigate up vs to the accounts page?

Reply:
Something to note about this as its currently implemented though is that it will navigate you to the new linked or created account after the send form closes. Not sure if this is the desired behavior.